app/src/main/java/com/amaze/filemanager/activities/TextEditorActivity.java

```diff
@@ -293,7 +295,7 @@ private void load() {
                     try {
                         mInput.setText(data.fileContents);
 
-                        if (mFile.hybridFileParcelable.getPath().contains(getExternalCacheDir().getPath())
+                        if (mFile.hybridFileParcelable != null && mFile.hybridFileParcelable.getPath().contains(getExternalCacheDir().getPath())
```

Comment:
Sorry for the late respose.
Remember that `getExternalCacheDir()`
>May return null if shared storage is not currently available.

--[Android Deveplopers - getExternalCacheDir()](https://developer.android.com/reference/android/content/Context.html#getExternalCacheDir())